VideoWeb/VideoWeb/ClientApp/src/typings.d.ts

```diff
@@ -27,9 +33,11 @@ declare interface PexipClient {
     setParticipantSpotlight(participantId: string, spotlight: boolean);
     setMuteAllGuests(mute: boolean);
     muteAudio(): boolean;
+    muteVideo(): boolean;
     setBuzz();
     clearBuzz(uuid?: string);
     clearAllBuzz(): () => void;
+    createAudioContext(stream: MediaStream): () => void;
```

Comment:
missing implementation?

Reply:
Thanks, this needs to go. This was just improving the typinggs but it's in the wrong place and not needed yet